data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_eth_v1_beacon_rewards_attestations_{epoch}.json

```diff
@@ -1,85 +1,94 @@
 {
-  "post": {
-    "tags": [
-      "Beacon",
-      "Rewards"
-    ],
-    "operationId": "getAttestationsRewards",
-    "summary": "Get attestations rewards",
-    "description": "Retrieve attestation reward info for validators specified by array of public keys or validator index. If no array is provided, return reward info for every validator.",
-    "parameters": [
-      {
-        "name": "epoch",
-        "required": true,
-        "in": "path",
-        "schema": {
-          "type": "string",
-          "description": "`uint64` Epoch number to query.",
-          "example": "1",
-          "format": "uint64"
-        }
+  "post" : {
+    "tags" : [ "Beacon", "Rewards" ],
+    "operationId" : "getAttestationsRewards",
+    "summary" : "Get attestations rewards",
+    "description" : "Retrieve attestation reward info for validators specified by array of public keys or validator index. If no array is provided, return reward info for every validator.",
+    "parameters" : [ {
+      "name" : "epoch",
+      "required" : true,
+      "in" : "path",
+      "schema" : {
+        "type" : "string",
+        "description" : "`uint64` Epoch number to query.",
+        "example" : "1",
+        "format" : "uint64"
       }
-    ],
-    "requestBody": {
-      "required": false,
-      "content": {
-        "application/json": {
-          "schema": {
-            "type": "array",
-            "items": {
-              "type": "string"
+    } ],
+    "requestBody" : {
+      "required" : false,
+      "content" : {
+        "application/json" : {
+          "schema" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
```

Comment:
this one still has that formatting from manually doing things i think...

Reply:
double checked, it's the same as after "but was...". And every json formatted like this